Makefile

```diff
@@ -344,12 +382,23 @@ stop-zkprover: ## Stops zkprover
 run-l1-explorer: ## Runs L1 blockscan explorer 
 	$(RUNEXPLORERL1DB)
 	$(RUNEXPLORERL1)
+	$(RUNEXPLORERFRONTENDL1)
+	$(RUNEXPLORERSTATSDBL1)
+	$(RUNEXPLORERSTATSL1)
+	$(RUNEXPLORERSMARTCONTRACTVERIFIERL1)
+	$(RUNEXPLORERVISUALIZERL1)
+	$(RUNEXPLORERSIGPROVIDERL1)
```

Comment:
if those are shared services, we should extract them to their own section

Reply:
Done